### PR TITLE
Branch rename in conf.py

### PR DESCRIPTION
**Pull request by robertodr**
Last step, I think, before renaming the default branch to `main` in the "Settings" tab.

### CODE REVIEW

File: content/conf.py [hunk 1]
```diff
@@ -32,64 +32,72 @@
...
+}
 
 # Intersphinx mapping.  For example, with this you can use
 # :py:mod:`multiprocessing` to link straight to the Python docs of that module.
 # List all available references:
 #   python -msphinx.ext.intersphinx https://docs.python.org/3/objects.inv
-#extensions.append('sphinx.ext.intersphinx')
-#intersphinx_mapping = {
+# extensions.append('sphinx.ext.intersphinx')
+# intersphinx_mapping = {
```
Comment (rkdarst) [inline content/conf.py:95]: My general convention is "stuff you might want to comment out does not have a space between comment and code.  But I guess this was auto-formatted, which doesn't accept subtlty?

**Review by rkdarst — APPROVED** [review]
Great!

I guess the plan is "anyone that generates from now on will get the new name, anyone from the past keeps the old name, so there is no reason to add auto-detection of things".  Or for that matter, even documentation about it.

So I see no reason to delay.  Can anyone else think of usability issues?

File: content/conf.py (hunk 1 above)
Comment (robertodr) [reply content/conf.py:95]: Yes, this is Black doing its autoformatting.

**Comment by rkdarst** [comment]
Let's go, no need to delay progress by talk.  Will you change the default branch name in the settings?

**Pull request merged**